tf-aws/provider.tf

```
@@ -0,0 +1,36 @@
+variable "region" {}
+
+# data "aws_eks_cluster" "eks" {
+#   name = "prod"
+# }
+#
+# data "aws_eks_cluster_auth" "eks" {
+#   name = "prod"
+# }
+
+# provider "helm" {
+#   kubernetes {
+#     host                   = data.aws_eks_cluster.eks.endpoint
+#     cluster_ca_certificate = base64decode(data.aws_eks_cluster.eks.certificate_authority.0.data)
+#     token = data.aws_eks_cluster_auth.eks.token
+#   }
+# }
+
+provider "aws" {
+  region = var.region
+}
+
```

```suggestion
    awscc = {
      source  = "hashicorp/awscc"
      version = "~> 0.1.0"
    }
```
i think need add this for your eks iam to work

---

nah  dont need